includes/wp-admin/class-settings.php

```diff
@@ -417,4 +461,49 @@ public static function add_settings_help_tab() {
 			'<p>' . \__( '<a href="https://github.com/automattic/wordpress-activitypub/issues">Report an issue</a>', 'activitypub' ) . '</p>'
 		);
 	}
+
+	/**
+	 * Add screen option.
+	 *
+	 * @param string $screen_settings The screen settings.
+	 * @param object $screen          The screen object.
+	 *
+	 * @return string The screen settings.
+	 */
+	public static function add_screen_option( $screen_settings, $screen ) {
+		if ( 'settings_page_activitypub' !== $screen->id ) {
+			return $screen_settings;
+		}
+
+		if ( isset( $_GET['welcome'] ) ) {
+			$welcome_checked = empty( $_GET['welcome'] ) ? 0 : 1;
+			\update_user_meta( \get_current_user_id(), 'activitypub_show_welcome_tab', $welcome_checked );
+		}
+
+		if ( isset( $_POST['activitypub_show_welcome_tab'] ) && isset( $_POST['screenoptionnonce'] ) ) {
+			$nonce   = sanitize_text_field( wp_unslash( $_POST['screenoptionnonce'] ) );
+			$welcome = sanitize_text_field( wp_unslash( $_POST['activitypub_show_welcome_tab'] ) );
+			// Verify screen options nonce.
+			if ( \wp_verify_nonce( $nonce, 'screen-options-nonce' ) ) {
+				$welcome_checked = empty( $welcome ) ? 0 : 1;
+				\update_user_meta( \get_current_user_id(), 'activitypub_show_welcome_tab', $welcome_checked );
+			}
+		}
+
+		$screen_settings = '<fieldset>
+		<legend class="screen-layout">' . __( 'Settings Pages', 'activitypub' ) . '</legend>
```

Comment:
```suggestion
		<legend class="screen-layout">' . \esc_html__( 'Settings Pages', 'activitypub' ) . '</legend>
```
This and the following translation functions should be escaped